_posts/2018-02-28-Didacta.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Die Didacta – eine erfolgreiche Messewoche"
+date: 2018-02-28
+author: Gina
+abstract: "Kurzer Beitrag darüber, was auf der Didacta Bildungsmesse in Hannover passiert ist."
```

Comment:
Eine Erfolgreiche Messewoche liegt hinter uns. In diesem Jahr präsentierten wir uns auf der Didacta als Teil des EDU Startup-Valley.